content/eng-hiring/debrief.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Debrief meeting guide
+permalink: /eng-hiring/debrief/
+layout: layouts/page
+tags: eng-hiring
+eleventyNavigation: 
+  parent: eng-hiring-debrief
+  key: eng-hiring-debrief-home
+  order: 1
+  title: Debrief meeting guide
+sidenav: true
+sticky_sidenav: true
+subnav:
+  - text: Purpose
+    href: "#purpose"
+  - text: Before the meeting
+    href: "#before-the-meeting"
+  - text: Conversation outline
+    href: "#conversation-outline"
+  - text: After the meeting
+    href: "#after-the-meeting"
+---
+
+The goal of the debrief meeting is to discuss the evidence and data the interview panel was able to learn about a candidate during the interview process.
+
+{% include 'eng-hiring/unconscious-bias-warning.html' %}
+
+## Purpose
+
+The purpose of the debrief meeting is to:
+
+1. **Share and discuss interview evidence and data.** Each interview has a different purpose and goals, and collects data about different aspects of the candidate. When this data is combined, it creates a more full view of the candidate and allows us all to spot discrepancies in our understanding. Do not share your interpretations, thoughts, or judgments about the candidate’s actions until after you have presented the data and you are discussing your scoring.
+
+2. **Create a single, unified summary of a candidate, resolving any discrepancies in interview data.** This summary will be used by the 18F Engineering Hiring team to make a [final selection]({{ '../selection' | url }}).
```

Comment:
This link works, but gets flagged by a link-checker. could you add a trailing `/` after "selection" i.e. `../selection/`